force-app/main/default/classes/HierarchySettingsService_TEST.cls

```diff
@@ -187,14 +187,11 @@ public with sharing class HierarchySettingsService_TEST {
         );
 
         Map<String, String> singleValueSettingUpdatesByNameInvalid = new Map<String, String>{
-            UTIL_Namespace.StrTokenNSPrefix('Account_Processor_Invalid__c') => 'Updated Value'
+            'Account_Processor_Invalid__c' => 'Updated Value'
```

Comment:
The side effect is using the namespace token prefix in our tests. This is a tradeoff I feel is relevant at this stage, but worth the effort.